src/AdvApi32.Desktop/AdvApi32.cs

```diff
@@ -19,14 +19,169 @@ public static partial class AdvApi32
         /// because services and service groups share the same name space.
         /// </summary>
         public const string SC_GROUP_IDENTIFIER = "+";
-        public const uint INFINITE = 0xFFFFFFFF;
+        public const uint SERVICE_NO_CHANGE = 0xFFFFFFFF;
```

Comment:
I was gonna say `uint` -> `int` but it might be better to do all the conversions in the same review linked to #60 or to an equivalent issue.